themes/digital.gov/src/js/common/in-page-navigation.js

```diff
@@ -0,0 +1,18 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const relatedItems = document.querySelector(".content__related-items");
+  const inPageNav = document.querySelector(".usa-in-page-nav__nav");
+  const navItems = document.querySelectorAll("li.usa-in-page-nav__item:not(.usa-in-page-nav__item--sub-item)");
+
+  console.log(navItems.length);
+
+  if (!relatedItems) {
+    return;
+  }
+
+  if (navItems.length < 3) {
+    inPageNav.remove();
```

Comment:
We should consider hiding via `display: none` instead of removing once we roll this out to other pages.

Reply:
Update the code to `inPageNav.style.display = none` so that it is ready for future changes.